.storybook/Colors.js

```diff
@@ -29,14 +29,17 @@ const Card = props => (
 const Comparison = ({ keys }) => (
   <Flex wrap>
     {keys.map(key => (
-      <Box
-        w={[1/3, 1/4, 1/5]}
+      <Flex
         key={key}
-        mr={3}
-        mb={1}>
-        <Card name={key} color={theme.colors[key]} />
-        <Card name={key} color={legacyTheme.colors[key]} />
-      </Box>
+        w={[1, 1/2, null, 1/3, 1/4]}
```

Comment:
Does styled-system support 5 ranges now?

Reply:
It supports how ever many `breakpoints` we have defined in our theme. Right now we have four, and the first number is for the non-media-query-scoped value, e.g. "xs" or below the first breakpoint